core/src/main/kotlin/com/ritense/valtimo/autoconfiguration/JobAutoConfiguration.kt

```diff
@@ -0,0 +1,36 @@
+/*
+ * Copyright 2015-2023 Ritense BV, the Netherlands.
+ *
+ *  Licensed under EUPL, Version 1.2 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *  https://joinup.ec.europa.eu/collection/eupl/eupl-text-eupl-12
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" basis,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ *
+ */
+
+package com.ritense.valtimo.autoconfiguration
+
+import com.ritense.valtimo.JobService
+import com.ritense.valtimo.contract.annotation.ProcessBean
+import org.camunda.bpm.engine.ManagementService
+import org.camunda.bpm.engine.ProcessEngine
+import org.springframework.context.annotation.Bean
+import org.springframework.context.annotation.Configuration
+
+@Configuration
+class JobAutoConfiguration {
+
+    @Bean
+    @ProcessBean
+    fun jobService(managementService: ManagementService): JobService {
```

Comment:
Can you add a `@ConditionalOnMissingBean(...)`, here?
So that implementations can choose to override/extend this service

Reply:
fixed